examples/load/WorkersCompensation.csv

```diff
@@ -0,0 +1,2 @@
+code    ,name                ,state,description
+WC-Code1,WorkersCompensation1,IL   ,WC Description
```

Comment:
You have't got there yet, but for the purposes of the integration test you'll want to change the name to be the auto-updated name in the test:

`workersCompensation-ext-1`